bin/liquidator/src/main.rs

```diff
@@ -552,6 +562,26 @@ impl LiquidationState {
         .await;
 
         if let Err(err) = result.as_ref() {
+            if let Some((ti, ti_name)) = err.try_unwrap_oracle_error() {
+                if self
+                    .oracle_errors
+                    .had_too_many_errors(LiqErrorType::Liq, &ti, Instant::now())
+                    .is_some()
+                {
+                    println!(
+                        "{:?} recording oracle error for token {} {}",
+                        chrono::offset::Utc::now(),
+                        ti_name,
+                        ti
+                    );
```

Comment:
why does it log if had_too_many_errors() is true? shouldn't this output the error if there have not been too many oracle issues for that ti?